oe_theme.theme

```diff
@@ -1289,6 +1290,17 @@ function oe_theme_preprocess_paragraph__oe_social_media_follow(array &$variables
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_text_featured_media(array &$variables): void {
```

Comment:
Unfortunately, it is not just for the testing problem. In ecl twig's template, they removed '|raw' filter from the video variable. So HTML for video was with autoescaping. unfortunately just passing render array didn't work.
On 'components' page it works because the ui_pattern module automatically doing marking input data as 'safe' by applying markup objects.
For reproducing the issue as I understand you can use code in any template like that:
`
{{ pattern('text_featured_media', {
  'video': '<iframe id=\"videoplayerI-181645\" src=\"//ec.europa.eu/avservices/play.cfm?ref=I-181645\"></iframe>',
  'video_ratio': 'invalid',
}) }}`

Reply:
I see but there is no real case right now where these would be just sent as string or '#markup' instead of a field's renderable array, but we can leave it in case it is sent as '#markup' for any future case

Reply:
I agree with Andras here, we are not covering any specific case and we are opening the door for more injection security issues. I would say lets only add this when and if we need it and use one of Andras' suggestions on the tests.

Reply:
I've checked and indeed Andras's solution work. it is not very clear why simple #markup doesn't work but html_tag element type works.